.github/workflows/sync-rest.yml

```diff
@@ -177,10 +216,12 @@ jobs:
           echo "::set-output name=name::$branch"
 
       - name: Copy globally synced files to target repo
+        if: steps.pr-already-exists.outputs.result == 'false'
         run: |
           cp -a main/global-files/. track-repo/
 
       - name: Determine if repo is a tooling repo
+        if: steps.pr-already-exists.outputs.result == 'false'
```

Comment:
I've not added this check to the steps that check for the repo being a tooling or track repo, as `steps.is-tooling-repo.outputs.result == 'true'` will be false anyway due to the step where that result is used. The same goes for the last steps that all check for `steps.changes.outputs.changes == 'true'`, which will always be false due to the step to set that to true not being executed if the PR already exists